package.json

```diff
@@ -29,9 +30,12 @@
     "node": ">= 0.10"
   },
   "dependencies": {
+    "@types/bluebird": "~3.0.36",
+    "@types/express": "~4.0.34",
```

Comment:
Those must remain in `dependencies`. Otherwise the user would have to install them manually.